src/main/java/mediathek/config/StandardLocations.kt

```diff
@@ -79,6 +95,8 @@ object StandardLocations {
         val userHome = SystemUtils.USER_HOME
         val path = if (SystemUtils.IS_OS_MAC_OSX)
             Paths.get(userHome, "Downloads")
+        else if (!SystemUtils.IS_OS_WINDOWS)
```

Comment:
Why do you check for !WINDOWS when this should work only on Linux? IS_OS_LINUX is more appropriate here.

Reply:
Doesn't Mediathekview run on e.g. recent BSDs (like FreeBSD)?

I don't mind changing this check to Linux, as that's all I care for, but technically this code path would work on any Unix (including BSDs) that's not macOS, so I figured the current check would be the correct approach 🙂

Reply:
It used to run before JavaFX and other native libs were introduced that are not available on FreeBSD. And to be honest, no-one runs FreeBSD as a desktop OS. Therefore checks for LINUX are better and it may give debugging hints to our future selfs ;)

Reply:
👍🏿 I've change the code to check for Linux.